illumos-utils/src/running_zone.rs

```diff
@@ -144,7 +147,7 @@ impl RunningZone {
 
     /// Returns the filesystem path to the zone's root
     pub fn root(&self) -> String {
-        format!("{}/root", self.inner.zonepath.display())
+        format!("{}/root", self.inner.zonepath)
     }
```

Comment:
This can be a `Utf8PathBuf`:

```rust
    pub fn root(&self) -> Utf8PathBuf {
        self.inner.zonepath.join("root")
    }
```

Reply:
Done in https://github.com/oxidecomputer/omicron/pull/2988/commits/ced5dc92aa41c74da6a846aaa9cd59d20d9c7843